examples/assets_dbt_python/setup.py

```diff
@@ -23,6 +23,7 @@
         "dagster-duckdb-pandas",
         # packaging v22 has build compatibility issues with dbt as of 2022-12-07
         "packaging<22.0",
+        "croniter<1.4.0",  # https://github.com/dagster-io/dagster/pull/14811
```

Comment:
these were needed since i just added the tests that ensure the examples work against published packages 

im not sure it worth it, but if these all referenced a singular `expected_user_pins.txt` that occasionally gets an entry like this then gets reset to empty on publish could be nice to be able to point to